### PR TITLE
Fixed ansible-playbook command in bootstrap hypervisor section

### PR DESCRIPTION
**Pull request by rebtoor**
- fixed few validation issues

As a pull request owner and reviewers, I checked that:
- [x] Appropriate testing is done and actually running
- [x] Appropriate documentation exists and/or is up-to-date:
  - [x] README in the role
  - [x] Content of the docs/source is reflecting the changes

### CODE REVIEW

File: docs/source/quickstart/02_prepare_env.md [hunk 1]
```diff
@@ -75,30 +76,35 @@ hypervisors:
...
+    -e ansible_user=root \
+    -e cifmw_target_host=[hypervisor|hypervisors] \
+    docs/source/files/bootstrap-hypervisor.yml
 ```
 
 ### VXLAN
-Once you're finished with the basic bootstrap, and if you're having two hypervisors, you can consume the
-[bootstrap-vxlan.yml](../files/bootstrap-vxlan.yml):
+
+Once you're finished with the basic bootstrap, and if you're having two hypervisors, you can consume the `bootstrap-vxlan.yml`:
```
Comment (lewisdenny) [inline docs/source/quickstart/02_prepare_env.md:103]: After completing the initial bootstrap process, if you are deploying on two hypervisors, you can consume the 'bootstrap-vxlan.yml' playbook

**Review by lewisdenny — CHANGES_REQUESTED** [review]
This patch looks good to me, just because we are already changing the line I made a suggestion. :)

File: docs/source/quickstart/02_prepare_env.md (hunk 1 above)
Comment (cjeanner) [reply docs/source/quickstart/02_prepare_env.md:103]: @rebtoor care to address this? @lewisdenny's proposal sounds better.

Comment (rebtoor) [reply docs/source/quickstart/02_prepare_env.md:103]: Sure! Thanks @lewisdenny !

**Review by lewisdenny — APPROVED** [review]
/lgtm

Thanks!

**Review by raukadah — APPROVED** [review]
/approve

**Pull request merged**